### PR TITLE
Issue#23 Splitting development setup into system deps and local deployment

### PR DESCRIPTION
**Pull request by kthblmfld**
Minor change to README to capture a few required system dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ for configuring / running the UI server.
 
 ## Development
 
+####System Dependencies
```
Comment (devsatishm) [inline README.md]: Need a space -> #### System Dependencies

File: README.md [hunk 2]
```diff
@@ -45,6 +45,21 @@ for configuring / running the UI server.
...
+    python -V
+
+If Python is missing or an earlier version, install python3 and pip3. 
+
+PyYaml is required for running tests. Install it using pip3:
+
+    pip3 install pyyaml
+
+
+####Deploying to Localhost
```
Comment (devsatishm) [inline README.md]: Need a space -> #### Deploying to Localhost

Comment (kthblmfld) [reply README.md]: Added spaces to both subsection headers

**Comment by devsatishm** [comment]
LGTM.

**Pull request merged**